pull-requests/PullRequestTemplate.md

```diff
@@ -0,0 +1,44 @@
+Thank you for contributing.
+Before submitting this PR, please make sure:
+
+-------------------------------------
+* [ ] I have followed the guidelines in [Contributing as Developers](https://github.com/microsoft/code-with-engineering-playbook/blob/master/CONTRIBUTING.md).
```

Comment:
This can act as the default contributing guide and once the project has their own, the PR template for the project can be replaced specific to the project.